packages/blade/src/components/Form/FormLabel.tsx

```diff
@@ -125,7 +140,7 @@ const FormLabel = ({
 
   const Component = as;
   // only set 120px label when device is desktop
-  const width = isLabelLeftPositioned && isDesktop ? makeSize(size[120]) : 'auto';
+  const width = isLabelLeftPositioned && isDesktop ? makeSize(labelWidth[size]) : 'auto'; // TODO: use size token
```

Comment:
open todo?

Reply:
This is resolved. Already using size token now

Reply:
Removed comment